scripts/integration-test.sh

```diff
@@ -1,3 +1,5 @@
+#!/bin/bash
```

Comment:
Added this to make editors understand that this is a shell script. Also made it an executable, for some reason it wasn't.